draft-ietf-taps-interface.md

```diff
@@ -3070,6 +3088,45 @@ and for contributing text, e.g., on multicast.
 
 --- back
 
+# Implementation Mapping {#implmapping}
+
+The way the concepts from this abstract interface map into concrete APIs in a
+given language on a given platform largely depends on the features and norms of
+the language and the platform. Actions could be implemented as functions or
+method calls, for instance, and Events could be implemented via event queues,
+handler functions or classes, communicating sequential processes, or other
+asynchronous calling conventions.
+
+## Types
+
+The basic types mentioned in {{notation}} typically have natural
+correspondences in practical programming languages, perhaps constrained by
+implementation-specific limitations. For example:
+
+- An Integer can typically be represented in C by an `int` or `long`, subject
+  to the underlying platform's ranges for each. To accommodate special values,
+  a C function that returns a non-negative `int` on success may return -1 on
+  failure. In Python, such a function might return `None` or raise an
+  exception.
+- In C, a Tuple may be represented as a `struct` with one member for each of
+  the value types in the ordered grouping. In Python, by contrast, a Tuple may
+  be represented natively as a `tuple`, a sequence of dynamically-typed
+  elements.
+- A Collection may be represented as a `std::set` in C++ or as a `set` in
```

Comment:
We could mention time duration representation here (can be a strong type in C++, in C it could be milliseconds as an integer).